lib/portlayer/exec/container_cache.go

```diff
@@ -79,3 +117,109 @@ func (conCache *containerCache) Remove(idOrRef string) {
 	}
 
 }
+
+func isContainerID(id string) bool {
```

Comment:
Why we are checking the IDs and why using regex? We put those ids there in the first place right?